core/src/main/java/org/apache/iceberg/jdbc/JdbcUtil.java

```diff
@@ -303,6 +287,325 @@ public static Properties filterAndRemovePrefix(Map<String, String> properties, S
     return result;
   }
 
+  static String getTableOrViewSql(boolean table) {
+    String tableOrViewTableName = table ? CATALOG_TABLE_NAME : CATALOG_VIEW_NAME;
+    String tableOrViewNamespace = table ? TABLE_NAMESPACE : VIEW_NAMESPACE;
+    String tableOrViewName = table ? TABLE_NAME : VIEW_NAME;
+    return String.format(
+        GET_TABLE_OR_VIEW_SQL, tableOrViewTableName, tableOrViewNamespace, tableOrViewName);
+  }
+
+  private static String getTableOrViewNamespaceSql(boolean table) {
+    String tableOrViewTableName = table ? CATALOG_TABLE_NAME : CATALOG_VIEW_NAME;
+    String tableOrViewNamespace = table ? TABLE_NAMESPACE : VIEW_NAMESPACE;
+    return String.format(
+        GET_TABLE_OR_VIEW_NAMESPACE_SQL,
+        tableOrViewNamespace,
+        tableOrViewTableName,
+        tableOrViewNamespace,
+        tableOrViewNamespace);
+  }
+
+  public static String getTableNamespaceSql() {
```

Comment:
can we double check and update all the new `public` methods? I think these can be package private considering that previous declarations of SQL was package private.

Reply:
If we put `private`, I have to remove all corresponding tests in `TestJdbcUtil`. If we want to keep the tests, it should be package protected.
Another option is to set `private` and remove the tests, as it's already covered by `TestJdbcViewCatalog` and `TestJdbcCatalog`.

Reply:
I mentioned as package private (default access specifier, without any keyword). Since the test and code is in same package, it should be accessible.

Reply:
`public static String getTableNamespaceSql()` -> `static String getTableNamespaceSql()`

Reply:
Yeah, OK, that's what I meant as well 😄 Let me do that.

Reply:
I agree, most (if not all) new methods that were introduced should be private or package-private

Reply:
also I would probably remove the `get` prefix from all methods, as that doesn't add any value

Reply:
yes. These guidelines are documented here
https://iceberg.apache.org/contribute/#java-style-guidelines

But we need to add the info about new lines after a block (if it has extra statements) for that doc later.